### PR TITLE
rework null handling in FromProtoValueCodecs

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
improve handling of null (unset) values returned from gRPC API in SGv2 REST API service.

### CODE REVIEW

File: sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java [hunk 1]
```diff
@@ -293,86 +326,112 @@ public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
...
     }
   }
 
   protected static final class TimestampCodec extends FromProtoValueCodec {
     @Override
     public Object fromProtoValue(QueryOuterClass.Value value) {
-      return Instant.ofEpochMilli(value.getInt()).toString();
+      return (value.getInnerCase() == QueryOuterClass.Value.InnerCase.NULL)
+          ? null
+          : Instant.ofEpochMilli(value.getInt()).toString();
```
Comment (jeffreyscarpenter) [inline sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java:363]: @mpenick we've been trying to use the operations on `io.stargate.grpc.Values` where possible but there is no `Values.timestamp() operation. Should there be?

Comment (mpenick) [reply sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java:363]: It's encoded in the `int` field: https://github.com/stargate/stargate/blob/v1.0.45/grpc-proto/proto/query.proto#L93-L94

Comment (jeffreyscarpenter) [reply sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java:363]: agree, but what I'm saying is there are operations on `Values`corresponding to other types that can be encoded in `int`:

- `Values.int_()`
- `Values.smallint()`
- `Values.tinyint()`

**Review by olim7t — APPROVED** [review]

**Comment by tatu-at-datastax** [comment]
Looks good, makes sense.

**Pull request merged**